src/docs/sphinx/Actions/Scenes.rst

```diff
@@ -402,6 +402,63 @@ Now we add a second render to the same example using every available parameter:
         near_plane: 0.1
         far_plane: 100.1
 
+Rendering Camera Configuration
+------------------------------
+Ascent supports two primary methods for defining the rendering camera:
+the Ascent native (VTKm style) camera, which is camera-centric, and the VisIt style camera, which is view-centric.
+Both formats provide control over how scenes are rendered, but they differ in terminology, orientation, and internal computation of the view matrix.
+
+Ascent Native (VTKm Style) Camera
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The native Ascent camera follows the VTKm camera model which focuses on defining the camera's physical parameters and orientation in 3D space.
+This camera-centric approach specifies where the camera is located, what it is looking at, and how it is oriented.
+The following parameters are supported:
+
+- ``2d`` : Enables 2D rendering mode.
+- ``look_at`` : A 3-element array [x, y, z] indicating the point the camera is directed toward.
+- ``position`` : A 3-element array [x, y, z] indicating the camera's position in space.
+- ``up`` : A 3-element vector that defines the up direction of the camera.
+- ``fov`` : Field of view in degrees for perspective projection.
+- ``xpan`` : Pan offset in the horizontal direction.
+- ``ypan`` : Pan offset in the vertical direction.
+- ``zoom`` : Uniform zoom factor applied to the view.
+- ``near_plane`` : Distance from the camera to the near clipping plane.
+- ``far_plane`` : Distance from the camera to the far clipping plane.
+- ``azimuth`` : Rotates the camera around the up axis.
+- ``elevation`` : Rotates the camera around the right axis.
+
+VisIt Style Camera
+^^^^^^^^^^^^^^^^^^
+Alternatively, the camera can be defined using the VisIt camera parameters which follows a view centric approach. 
+Instead of positioning the camera explicitly, it defines the view using vectors like ``view_normal`` and ``view_up``, which describe the direction and orientation of the camera relative to a focal point.
+The following parameters are supported:
+
+- ``view_normal`` : A vector [x, y, z] indicating the direction from the camera toward the scene.
```

Comment:
Like [we discussed before](https://github.com/Alpine-DAV/ascent/pull/1596/files#r2292040412), in VisIt the view normal is pointing towards the camera from the focus, not from the camera toward the scene.